content/en/logs/guide/azure-automated-log-forwarding.md

```diff
@@ -57,6 +73,61 @@ The sections below provide instructions for completing each page of the template
 1. Review the finalized deployment details.
 2. Click **Create**.
 
+## Architecture
+
+### Services used
+
+1. [Azure Function][15] apps are used to discover resources in your Azure subscriptions, scale log forwarders, and configure diagnostic settings on the detected resources. 
+2. [Azure Container Apps][8] are used to collect resource logs generated by diagnostic settings, track which logs have been processed already, and submit them to Datadog.
+3. [Azure Storage Accounts][9] are used to store logs generated by your resources, as well as a small cache of metadata such as subscription IDs, resource IDs, and regions.
```

Comment:
```suggestion
- [Azure Function][15] apps are used to discover resources in your Azure subscriptions, scale log forwarders, and configure diagnostic settings on the detected resources. 
- [Azure Container Apps][8] are used to collect resource logs generated by diagnostic settings, track which logs have been processed already, and submit them to Datadog.
- [Azure Storage Accounts][9] are used to store logs generated by your resources, as well as a small cache of metadata such as subscription IDs, resource IDs, and regions.
```

Since there's no hierarchy to this list, it should be unordered